cmd/root.go

```diff
@@ -56,7 +56,32 @@ func NewRootCmd() *cobra.Command {
 	rootCmd := &cobra.Command{
 		Use:   "astro",
 		Short: "Run Apache Airflow locally and interact with Astronomer",
-		Long:  "Welcome to the Astro CLI. Astro is the modern command line interface for data orchestration. You can use it for Astro, Astronomer Software, or local development.",
+		Long: `
+                                          _..._
+                                        .'     '.      _
+                                       /    .-""-\   _/ \
+                                     .-|   /:.   |  |   |
+                                     |  \  |:.   /.-'-./
+                                     | .-'-;:__.'    =/
+                                     .'=  *=ASTRO _.='
+                                    /   _.  |    ;
+                                   ;-.-'|    \   |
+                                  /   | \    _\  _\
+                                  \__/'._;.  ==' ==\
+                                           \    \   |
+                                           /    /   /
+                                           /-._/-._/
+                                           \   '\  \
+                                            '-._/._/
+ ________   ______   _________  ______    ______             ______   __        ________
+/_______/\ /_____/\ /________/\/_____/\  /_____/\           /_____/\ /_/\      /_______/\
+\::: _  \ \\::::_\/_\__.::.__\/\:::_ \ \ \:::_ \ \   _______\:::__\/ \:\ \     \__.::._\/
+ \::(_)  \ \\:\/___/\  \::\ \   \:(_) ) )_\:\ \ \ \ /______/\\:\ \  __\:\ \       \::\ \
+  \:: __  \ \\_::._\:\  \::\ \   \: __ '\ \\:\ \ \ \\__::::\/ \:\ \/_/\\:\ \____  _\::\ \__
+   \:.\ \  \ \ /____\:\  \::\ \   \ \ '\ \ \\:\_\ \ \          \:\_\ \ \\:\/___/\/__\::\__/\
+    \__\/\__\/ \_____\/   \__\/    \_\/ \_\/ \_____\/           \_____\/ \_____\/\________\/
+
+Welcome to the Astro CLI. Astro is the modern command line interface for data orchestration. You can use it for Astro, Astronomer Software, or local development.`,
```

Comment:
```suggestion
Welcome to the Astro CLI, the modern command line interface for data orchestration. You can use it for Astro, Astronomer Software, or local development.`,
```
One minor edit to make it clear that "Astro" is not the CLI but the product that you can use with the CLI